test/parameters.jl

```diff
@@ -49,6 +49,20 @@ pdiagmat(args...) = PDiagMat(args...)
         )
     end
 
+    @testset "orthogonal" begin
+        is_almost_orthogonal(X::AbstractMatrix, tol) = norm(X'X - I) < tol
+        @testset "nearest_orthogonal_matrix" begin
+            X_orth = ParameterHandling.nearest_orthogonal_matrix(randn(5, 4))
+            @test is_almost_orthogonal(X_orth, 1e-9)
+            X_orth_2 = ParameterHandling.nearest_orthogonal_matrix(X_orth)
+            @test X_orth ≈ X_orth_2 # nearest_orthogonal_matrix is a projection.
```

Comment:
they reason they are not exactly `==` is just numeric noise? What do you want to imply by saying that it's a "projection"?

Reply:
> they reason they are not exactly == is just numeric noise?

Yes :)

> What do you want to imply by saying that it's a "projection"?

I mean that applying the function twice yields the same result as applying it once, see e.g. https://en.wikipedia.org/wiki/Projection_(linear_algebra)

Reply:
I see, thanks!